Applications/MaterialManager/Contracts/Material/Edgebands/Interfaces/IMaterialManagerClientMaterialEdgebands.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace HomagConnect.MaterialManager.Contracts.Material.Edgebands.Interfaces
+{
+    /// <summary>
+    /// Interface for MaterialManager Edgebands Client.
+    /// </summary>
+    public interface IMaterialManagerClientMaterialEdgebands
```

Comment:
sure about the naming? 2 time Material.
Maybe: IMaterialManagerClientEdgebands

Reply:
@HOMAG-Boris-Wehrle suggested this from the start
I think the background is that there will be processing and the client for that will be called
IMaterialManagerClientProcessingOffcutParameters

Reply:
It is okay, just wanted to be sure that it is intended.